### PR TITLE
added cart page component.

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added quantity field in the card on the cart page.
- given card shadow on the cart page.

image

### CODE REVIEW

File: components/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+.cart-right-section{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.cart-price-heading{
+    color: var(--primary-color);
+    font-size: 2rem;
+    font-weight: 500;
+    /* width: 40rem; */
```
Comment (rutu14) [inline components/cart/cart.css]: Remove the comments if not in use

**Review by rutu14 — APPROVED** [review]

File: components/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/* cart */
+.display-cart-section{
+    display: flex;
+    flex-direction: row;
+    max-width: 100%;
+    margin: 3rem 2rem; 
+    padding: 3rem;
+}
+.grid-3-column-layout-cart{
```
Comment (itsmebipul00) [inline components/cart/cart.css:9]: it seems like here you are trying to make a 3 column layout using grid but I can see only 1 column of width 3fr made, to make a 3 column you could use g-tem-colms: 1fr 1fr 1fr , defining width for each column or g-temlate-colms: repeat(3, 1fr)

Comment (itsmebipul00) [reply components/cart/cart.css:9]: Or if it is just a single colum layout you could change the classname

Comment (itsmebipul00) [reply components/cart/cart.css:9]: What you have done here is made 1 colum with width 3fr and 1 row with height1fr

File: components/cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+.grid-1-column-layout{
+    display: grid;
+    grid-template-columns: 1fr;
+    justify-items: right;
+    margin:5rem 3rem 3rem 8rem;
+    padding: 1rem;
+}
+.cart-img{
+    height: 40rem;
+    /* height: 35rem; */
```
Comment (zoef-yt) [inline components/cart/cart.css]: Remove commented code

File: components/cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    justify-content: space-around;
+}
+.cart-price-heading{
+    color: var(--primary-color);
+    font-size: 2rem;
+    font-weight: 500;
+    /* width: 40rem; */
+}
+.cart-total-section{
+    background-color: white;
```
Comment (zoef-yt) [inline components/cart/cart.css]: Use var for colours

**Review by zoef-yt — APPROVED** [review]
Nice code. Easy to understand. Great UI. Good to merge with the changes.

**Comment by zoef-yt** [comment]
Make sure to right description in the pr and add images or gif

File: components/cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    border-radius: 5px;
+    box-shadow: 0px 0px 10px 1px var(--navigation-container-shadow);
+    max-width: 100%;
+    padding: 3rem;
+}
+.cart-total{
+    display: flex;
+    flex-direction: column;
+}
+.total-cost, .discount, .coupon, .convenience-fee, .total-amount{
```
Comment (purvasharmadev) [inline components/cart/cart.css:52]: Try creating a single class for styles that need to be applied to more than 3 elements this helps in refactoring. Just a suggestion else everything looks good to go.

**Review by purvasharmadev — APPROVED** [review]

**Review by itsmebipul00 — APPROVED** [review]

File: components/cart/cart.css (hunk 2 above)
Comment (zoyatanweer) [reply components/cart/cart.css:9]: ok I will look into it.

File: components/cart/cart.css (hunk 5 above)
Comment (zoyatanweer) [reply components/cart/cart.css:52]: ok Purva will keep that in mind.

File: components/cart/cart.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    flex-direction: row;
+    justify-content: space-around;
+}
+.cart-price-heading{
+    color: var(--primary-color);
+    font-size: 2rem;
+    font-weight: 500;
+}
+.cart-total-section{
+    background-color: #ffffff;
```
Comment (zoef-yt) [inline components/cart/cart.css]: use var for colour

File: css/main.css [hunk 7]
```diff
@@ -191,6 +192,40 @@ ul{
     left: 200px;
     right: 13px;
```
Comment (zoef-yt) [inline css/main.css]: use rem

File: css/main.css [hunk 8]
```diff
@@ -191,6 +192,40 @@ ul{
...
+}
+.qty-number{
+    border: 1px solid var(--primary-color);
+    border-radius: 5px;
+    font-size: 1.5rem;
+    height: 40%;
+    width:20%;
+    text-align: center;
+    /* margin-bottom: 15px; */
+    /* padding: 5px;  */
```
Comment (zoef-yt) [inline css/main.css:213]: remove commented code.

File: css/main.css [hunk 9]
```diff
@@ -191,6 +192,40 @@ ul{
...
+/* cart */
+.img-horizontal{
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+}
+.qty-product{
+    color: var(--primary-color);
+    width: 100%;
+    margin:3px;
+    /* height: 100%; */
```
Comment (zoef-yt) [inline css/main.css]: remove commented code.

File: css/main.css [hunk 10]
```diff
@@ -162,6 +162,7 @@ ul{
     height: 3rem;
     width: 3rem;
     margin-right: 2.5rem;
+    /* fill: var(--primary-color); */
```
Comment (zoef-yt) [inline css/main.css]: remove commented code.

File: components/signup/signup.html [hunk 11]
```diff
@@ -15,17 +15,31 @@
...
+                        <!-- <button class="btn-icon btn-icon-wishlist">4</button> -->
+                    </a>
+                </div>
+                <div class="single-action-btn">
+                    <a href="../cart/cart.html">
+                        <svg class="right-nav-img" xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"/></svg>
+                        <!-- <button class="btn-icon btn-icon-cart">3</button> -->
```
Comment (zoef-yt) [inline components/signup/signup.html:40]: remove commented code.

File: components/signup/signup.html [hunk 12]
```diff
@@ -15,17 +15,31 @@
...
+                    </a>
+                </div>
+                <div class="single-action-btn">
+                    <a href="../wishlist/wishlist.html">
+                        <svg class="right-nav-img" xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z"/></svg>
+                        <!-- <button class="btn-icon btn-icon-wishlist">4</button> -->
```
Comment (zoef-yt) [inline components/signup/signup.html:34]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Pull request merged**